config/unicorn.rb

```diff
@@ -0,0 +1,41 @@
+app_path = File.expand_path('../../', __FILE__)
```

Comment:
> __[RuboCop]__ Style/ExpandPathArguments: Use `expand_path('..', __dir__)` instead of `expand_path('../../', __FILE__)`. ([link](https://sider.review/gh/repos/169721588/pulls/26#issue-3760813))

aws カリキュラムにしたがって記述